Functions/xASL_bids_PrepareDRO.m

```diff
@@ -0,0 +1,96 @@
+function xASL_bids_PrepareDRO(droTestPatient,droSubject)
+%xASL_bids_PrepareDRO Prepare DRO test patient for BIDS2Legacy conversion.
+%
+% FORMAT: xASL_bids_PrepareDRO(droTestPatient,droSubject)
+% 
+% INPUT:
+%   droTestPatient      - Path to the DRO (CHAR ARRAY, REQUIRED)
```

Comment:
You probably needs to give more details of what exactly are the inputs - names and structure of the files and what exactly the function does with it.

Reply:
I added more descriptive text in 3e2b8e1.